Jenkinsfile

```diff
@@ -461,30 +430,20 @@ pipeline {
 
         stage("Full tests I"){
             parallel{
-                stage('Int8 Hip Release All') {
-                    agent{ label rocmnode("vega20") }
+
+                stage('GCC codecov') {
+                    agent{ label rocmnode("vega") }
                     steps{
-                        buildJob('hcc', flags: '-DMIOPEN_TEST_INT8=On -DBUILD_DEV=On -DMIOPEN_TEST_ALL=On -DCMAKE_BUILD_TYPE=release', image: image+"rocm", prefixpath: '/opt/rocm', gpu_arch: "gfx906")
+                        buildJob('g++-5', flags: '-DBUILD_DEV=On -DCMAKE_BUILD_TYPE=debug', codecov: true, gpu_arch: "gfx900;gfx906")
                     }
                 }
-
-                stage('Bfloat16 Hip Release All') {
```

Comment:
[todo later] This `Bfloat16 Hip Release All` does not execute any tests. Fix and restore.

Reply:
![image](https://user-images.githubusercontent.com/12697412/99197305-c3e9d980-27a2-11eb-98a0-ebe089e1889f.png)